### ISSUE TITLE
need short info about modules

### ISSUE THREAD

**Issue by jogi01** [ForgeEssentials/ForgeEssentials#2122]
hey there :)

After some time i am planning a new server with your great essentials as "backend".
Now i am configuring and i need some description about..
..what is factions, mapper and perftools ?

Furthermore what is "item permission manager" which is switchable in main.cfg ?

Thank you in advance and respect for your great work !

regards

Jogi

**Comment by spacebuilder2020** [comment]
factions is incomplete and buggy.  Don't use it.  Mapper creates a Image of the current region and saves it to region.png, and  Perf tools provides basic performance information.  As for "item permission manager", I am not entirely sure what it does cause I have not played with it much.  You will have to ask @olee or @luacs1998.

**Comment by olee** [comment]
Actually factions is not buggy, it's just not finished.
But it can be used to allow users to join some groups themselves (though if you want factions to have special permissions, you will have to set them yourself for now).

Mapper is a part of remote module and basically a lightweight dynmap module.
It can be viewed via remote panel for example.

**Comment by spacebuilder2020** [comment]
Well, the main thing I disliked about factions was the creation of extra groups by unsanctioned users.  This resulted in players leaving the default _GUESTS_ group.  In addition, when they left the faction, it did not put them back in the default group.
But I just realized I fixed that in https://github.com/ForgeEssentials/ForgeEssentials/commit/7b4f4b135dc479d123c7d9d728f1ccfec814b709.
So factions may be fine, but I still would not recommend them if you have a better option from another mod.  Same with plots.
